src/main/scala/org/scalacheck/ops/time/JavaTimeGenerators.scala

```diff
@@ -0,0 +1,46 @@
+package org.scalacheck.ops.time
+
+import java.time.temporal._
+import java.time.{Duration, Instant}
+
+object JavaTimeGenerators extends JavaTimeGenerators
+trait JavaTimeGenerators extends AbstractDateTimeGenerators {
+  override type DateTimeType = Instant
+  override type DurationType = TemporalAmount
+  override type ParamsType = JavaTimeParams
```

Comment:
I was wondering how the heck are you able to override methods and provide a specific implicit param only to figure its defined as a type here :D